app/application/adapter.js

```diff
@@ -0,0 +1 @@
+export { default } from 'ember-local-storage/adapters/local';
```

Comment:
General perfect 👌 But please change for avoid future problems with `mirage.js` in test environment which we had in our 'Saturday app':
```suggestion
import RESTAdapter from '@ember-data/adapter/rest';
import LocalStorageAdapter from 'ember-local-storage/adapters/local';
import ENV from 'mms-ember-project/config/environment';

const isTesting = ENV.environment === 'test';

const Adapter = isTesting ? RESTAdapter : LocalStorageAdapter;

export default class ApplicationAdapter extends Adapter {}

```